### ISSUE TITLE
Design & discovery: Hazard stories for MVP

### ISSUE THREAD

**Issue by janelx** [weather-gov/weather.gov#148]
### Design & discovery: Hazard stories for MVP
[Preliminary MVP flow (in Figma)](https://www.figma.com/file/FKZbKMvnwMCOahfEIH7wfJ/sprint-2.5-%2F-MVP-flows?type=whiteboard&node-id=0%3A1&t=k5oWuoesqnI0qAl7-1)
[MVP user stories - design details](https://docs.google.com/document/d/1rtjpIUXBg39E44aVKJNjH8fTRBOPIHszAxhRbODLH6c/edit)

**Supporting user flow:** Person clicks on a link from an NWS alert

**Stories to include**
- 4.1 As a member of the public who got a hazard alert through their weather app, I want to click a link and see the full WWA hazard
- 4.2 … I want to be made aware of any hazards for a specific location
- 4.3 … I want to understand at-a-glance what, where, and how bad the threat/impact level of a hazard for a 4.4 specific location
- 4.4 … I would like to know what to expect for the next 24 hours (could be more?) for a specific location
- 4.5… I want to understand the actions or steps I could take to save my life or property at my location
- 4.6… I'd like to know when and where the hazard is in order to see if the hazard is approaching me or near me/specific location

Related:
- As an external user, I want to be made aware of the of any hazards for a specific location
**Added just to capture broader need for the following 2 stories. 

## Tasks
- Design and define molecules/layout
- Collab w/content for hierarchy of content and content details
- PO collab and review for organizational needs and feasibility
- Feasibility collab and review w/eng
- Determine if an user research is needed

## By the end of this sprint
- Designs ready for review: molecules and layout designed for each story supporting flow for hazard (link from outside source to full alert)
- Beginning documentation of acceptance criteria and supporting need for each user story/molecule
-- Wireframe and needed support (technical and organizational)
-- Documentation of high level key implementation details

**Continuous discovery**
- What we might want to test with users and what we have less certainty about

Out of scope - future sprint:
- Secondary visual imagery discovery for hazards

**Comment by colinmurphy01** [comment]
I believe this ticket is in review:
- Hazard SMEs and Champions on Wednesday, Sept 13
- Forecaster feedback

Once we get feedback, we can then finalize our definition for the MVP Hazard flow.

**Comment by colinmurphy01** [comment]
@janelx are we still waiting on forecaster feedback on this? I don't think we've gotten it, but Sarah might have this prioritized against partner/public needs as well.

**Comment by janelx** [comment]
@colinmurphy01 I believe we have received enough in from forecasters to begin creating this. I have a good idea of what it will look like and how we might be able to begin testing it. It sounds like there is a large public and partner need for hazard stories (forecaster communication around hazard) and the hazard page in general.

**Comment by janelx** [comment]
@colinmurphy01 the comment above is for #155  Hazard Forecasts, which I mistook this ticket for. I don't think this ticket requires forecaster feedback at the moment? I'm not sure where that need came from. I created a new ticket for a more detailed wireframe #242. I can review flow and components with POs this Friday and if approved we can close this ticket.

**Comment by colinmurphy01** [comment]
Using #160 to track wireframing work and closing this.
